test/hca-rjsf/config/medicare.unit.spec.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester, submitForm } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/hca-rjsf/config/form';
+
+describe('Hca medicare', () => {
+  const { schema, uiSchema } = formConfig.chapters.insuranceInformation.pages.medicare;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('input').length)
+      .to.equal(4);
+  });
+
+  it('should not submit empty form', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          onSubmit={onSubmit}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    submitForm(form);
+
+    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(2);
+    expect(onSubmit.called).to.be.false;
+  });
+
+  it('should reveal required effective date', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          onSubmit={onSubmit}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('input, select').length).to.equal(4);
+
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_isEnrolledMedicarePartAYes'), {
+      target: {
+        value: 'Y'
+      }
+    });
+
+    expect(formDOM.querySelectorAll('input, select').length).to.equal(7);
+
+    submitForm(form);
+
+    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(2);
```

Comment:
Since this should just test for the appearance of the effective date validation error, how about answering the eligibility question and then checking for 1 error here.